core/lib/src/local/client.rs

```diff
@@ -114,11 +117,13 @@ impl Client {
     /// ```rust
     /// use rocket::local::Client;
     ///
-    /// let client = Client::new(rocket::ignite()).expect("valid rocket");
+    /// # let _ = async {
```

Comment:
I think this pattern in some of these tests - checking for compilation only and not running any code - is my only misgiving. I did make a conscious effort to only do this when there were no `assert!`s made in the test, but it could still somewhat reduce coverage. @SergioBenitez would you prefer for all of these to use `rocket::async_test` instead of `let _ =`, to ensure the code actually runs as well?

Reply:
Sure, if it's an easy change, why not?